### PR TITLE
Update genesis-contracts.md for Amoy

### PR DESCRIPTION
**Pull request by simonDos**

### CODE REVIEW

**Comment by 0xgraciegrace** [comment]
@simonDos let us know when this ready for @EmpieichO to approve.

**Comment by simonDos** [comment]
@0xsharma will greenlight this, once Amoy is ready.

File: docs/pos/reference/contracts/genesis-contracts.md [hunk 1]
```diff
@@ -84,5 +84,52 @@ Parent chain: Ethereum mainnet
 | Contracts             | Address                                    |
 |-----------------------|--------------------------------------------|
 | ChildChain            | [0xD9c7C4ED4B66858301D0cb28Cc88bf655Fe34861](https://polygonscan.com/address/0xD9c7C4ED4B66858301D0cb28Cc88bf655Fe34861) |
+
+
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/genesis-contracts.md]:
```suggestion
```

File: docs/pos/reference/contracts/genesis-contracts.md [hunk 2]
```diff
@@ -84,5 +84,51 @@ Parent chain: Ethereum mainnet
 | Contracts             | Address                                    |
 |-----------------------|--------------------------------------------|
 | ChildChain            | [0xD9c7C4ED4B66858301D0cb28Cc88bf655Fe34861](https://polygonscan.com/address/0xD9c7C4ED4B66858301D0cb28Cc88bf655Fe34861) |
+
+### Amoy
+
+**Parent chain: Sepolia**
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/genesis-contracts.md]:
```suggestion
Parent chain: Sepolia
```

File: docs/pos/reference/contracts/genesis-contracts.md [hunk 3]
```diff
@@ -84,5 +84,51 @@ Parent chain: Ethereum mainnet
...
+| ERC20Predicate        | [0x15EA6c538cF4b4A4f51999F433557285D5639820](https://sepolia.etherscan.io/address/0x15EA6c538cF4b4A4f51999F433557285D5639820) |
+| ERC721Predicate       | [0x0059bBF8E5b9b071acc7682B6fe198c32AAA2A97](https://sepolia.etherscan.io/address/0x0059bBF8E5b9b071acc7682B6fe198c32AAA2A97) |
+| EventsHubProxy        | [0x700e0f2AfBd92e2b3fF91CAD8C62A564690ddf39](https://sepolia.etherscan.io/address/0x700e0f2AfBd92e2b3fF91CAD8C62A564690ddf39) |
+| EIP1559Burn           | [0xeCDD77cE6f146cCf5dab707941d318Bd50eeD2C9](https://sepolia.etherscan.io/address/0xeCDD77cE6f146cCf5dab707941d318Bd50eeD2C9) |
+| **Tokens**            | |
+| MaticToken            | [0x3fd0A53F4Bf853985a95F4Eb3F9C9FDE1F8e2b53](https://sepolia.etherscan.io/address/0x3fd0A53F4Bf853985a95F4Eb3F9C9FDE1F8e2b53) |
+| MaticWeth             | [0x700dDE29De87ed2c01c27C896dc8Badb4f671302](https://sepolia.etherscan.io/address/0x700dDE29De87ed2c01c27C896dc8Badb4f671302) |
+| RootERC721            | [0x13B0Edd9312886Ac0C73116e767208bEd1199679](https://sepolia.etherscan.io/address/0x13B0Edd9312886Ac0C73116e767208bEd1199679) |
+
+
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/genesis-contracts.md]:
```suggestion
```

File: docs/pos/reference/contracts/genesis-contracts.md [hunk 4]
```diff
@@ -84,5 +84,50 @@ Parent chain: Ethereum mainnet
...
+| ERC20Predicate        | [0x15EA6c538cF4b4A4f51999F433557285D5639820](https://sepolia.etherscan.io/address/0x15EA6c538cF4b4A4f51999F433557285D5639820) |
+| ERC721Predicate       | [0x0059bBF8E5b9b071acc7682B6fe198c32AAA2A97](https://sepolia.etherscan.io/address/0x0059bBF8E5b9b071acc7682B6fe198c32AAA2A97) |
+| EventsHubProxy        | [0x700e0f2AfBd92e2b3fF91CAD8C62A564690ddf39](https://sepolia.etherscan.io/address/0x700e0f2AfBd92e2b3fF91CAD8C62A564690ddf39) |
+| EIP1559Burn           | [0xeCDD77cE6f146cCf5dab707941d318Bd50eeD2C9](https://sepolia.etherscan.io/address/0xeCDD77cE6f146cCf5dab707941d318Bd50eeD2C9) |
+| **Tokens**            | |
+| MaticToken            | [0x3fd0A53F4Bf853985a95F4Eb3F9C9FDE1F8e2b53](https://sepolia.etherscan.io/address/0x3fd0A53F4Bf853985a95F4Eb3F9C9FDE1F8e2b53) |
+| MaticWeth             | [0x700dDE29De87ed2c01c27C896dc8Badb4f671302](https://sepolia.etherscan.io/address/0x700dDE29De87ed2c01c27C896dc8Badb4f671302) |
+| RootERC721            | [0x13B0Edd9312886Ac0C73116e767208bEd1199679](https://sepolia.etherscan.io/address/0x13B0Edd9312886Ac0C73116e767208bEd1199679) |
+
+**Child Chain: Amoy**
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/genesis-contracts.md]:
```suggestion
Child chain: Amoy
```

File: docs/pos/reference/contracts/genesis-contracts.md [hunk 5]
```diff
@@ -84,5 +84,50 @@ Parent chain: Ethereum mainnet
...
+| Contracts             | Address                                    |
+|-----------------------|--------------------------------------------|
+| ChildChain            | [0x4f9cd8a945EE035523979D7A120a23999D17D8C0](https://amoy.polygonscan.com/address/0x4f9cd8a945EE035523979D7A120a23999D17D8C0/) |
+| EIP1559Burn           | [0xeCDD77cE6f146cCf5dab707941d318Bd50eeD2C9](https://amoy.polygonscan.com/address/0xeCDD77cE6f146cCf5dab707941d318Bd50eeD2C9/) |
+| **Tokens**            | |
+| MaticToken            | [0x0000000000000000000000000000000000001010](https://amoy.polygonscan.com/address/0x0000000000000000000000000000000000001010/) |
+| MaticWeth             | [0x41Dc3C8eB8368bd9139Cec50434a0C294c8c1102](https://amoy.polygonscan.com/address/0x41Dc3C8eB8368bd9139Cec50434a0C294c8c1102/) |
+| RootERC721            | [0x3ADBC484Ff0cFEb657e1A9AF8F3CB16DC0B53e7e](https://amoy.polygonscan.com/address/0x3ADBC484Ff0cFEb657e1A9AF8F3CB16DC0B53e7e/) |
+
 </TabItem>
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/genesis-contracts.md]:
```suggestion
```

File: docs/pos/reference/contracts/genesis-contracts.md [hunk 6]
```diff
@@ -84,5 +84,49 @@ Parent chain: Ethereum mainnet
...
+| Contracts             | Address                                    |
+|-----------------------|--------------------------------------------|
+| ChildChain            | [0x4f9cd8a945EE035523979D7A120a23999D17D8C0](https://amoy.polygonscan.com/address/0x4f9cd8a945EE035523979D7A120a23999D17D8C0/) |
+| EIP1559Burn           | [0xeCDD77cE6f146cCf5dab707941d318Bd50eeD2C9](https://amoy.polygonscan.com/address/0xeCDD77cE6f146cCf5dab707941d318Bd50eeD2C9/) |
+| **Tokens**            | |
+| MaticToken            | [0x0000000000000000000000000000000000001010](https://amoy.polygonscan.com/address/0x0000000000000000000000000000000000001010/) |
+| MaticWeth             | [0x41Dc3C8eB8368bd9139Cec50434a0C294c8c1102](https://amoy.polygonscan.com/address/0x41Dc3C8eB8368bd9139Cec50434a0C294c8c1102/) |
+| RootERC721            | [0x3ADBC484Ff0cFEb657e1A9AF8F3CB16DC0B53e7e](https://amoy.polygonscan.com/address/0x3ADBC484Ff0cFEb657e1A9AF8F3CB16DC0B53e7e/) |
+
+</Tabs>
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/genesis-contracts.md]:
```suggestion
```

**Review by 0xgraciegrace — APPROVED** [review]

**Comment by 0xgraciegrace** [comment]
@0xsharma @simonDos approving this update. Please let us know if something is not accurate.

**Review by 0xsharma — APPROVED** [review]

**Pull request merged**